### ISSUE TITLE
groups.google.com/forum nolonger publicly accessable

### ISSUE THREAD

**Issue by shnwnd** [endless-sky/endless-sky.github.io#23]
https://github.com/endless-sky/endless-sky.github.io/blob/496c8db1a85fbee26ffcbe063113f68cfd0fa372/_layouts/default.html#L130

This line probably ought to change. It doesn't seem publicly accessible any longer. I have heard of a discord, would that be an okay replacement?

**Comment by shnwnd** [comment]
This whole navigation section should be in a separate file. I don't think individual navigation line items shouldn't be defined in a layout file.

**Comment by MCOfficer** [comment]
Given that discord is easily the most active community, i'd say yes.

**Comment by tehhowch** [comment]
The discord is not official, and will not be referred from the official Endless Sky website.

Loading the forum works just fine for me. Have you signed into your Google account?
![image](https://user-images.githubusercontent.com/20871346/90318318-530df980-def5-11ea-8d06-db3f7a59c406.png)

**Comment by MCOfficer** [comment]
It works for me on this account, but not on another one. Notice the marked text.

![image](https://user-images.githubusercontent.com/22377202/90320542-83c85000-df42-11ea-99c7-2fce3e8c2634.png)


![5eMIdg2k7g](https://user-images.githubusercontent.com/22377202/90320577-c0944700-df42-11ea-9082-afe430f3377f.png)
The text roughly translates to "this content is not available - check if the link is correct or switch account".

**Comment by MCOfficer** [comment]
> The discord is not official, and will not be referred from the official Endless Sky website.

Gotta ask the question, what would make the discord official? MZ's participation? Every active developer and content creator is there, which is much more than steam or google groups can claim.

**Comment by shnwnd** [comment]
In an incognito chrome browser (logged out) I get 
![Signed out](https://user-images.githubusercontent.com/16008819/90461125-51d9f980-e0ba-11ea-9abe-f750820e45ed.png)

![Signed in](https://user-images.githubusercontent.com/16008819/90461330-d62c7c80-e0ba-11ea-8a74-19d10b659826.png)

**Comment by samrocketman** [comment]
I also get not available and I'm signed into my Google account.  I get the same error whether logged in or incognito.

![Screenshot from 2020-11-28 23-08-03](https://user-images.githubusercontent.com/875669/100533116-c69af480-31ce-11eb-9f6b-2781b0f7cc62.png)

**Comment by samrocketman** [comment]
Update: Google groups still broken for ES project.

**Comment by MCOfficer** [comment]
Yes, we know. Until MZ reconfigures it *somehow*, this will have to remain open.

**Comment by Amazinite** [comment]
https://groups.google.com/g/endless-sky This is the URL I have when I go to the Google group, instead of URL shown in the issue description. Does this one work?

**Comment by samrocketman** [comment]
@Amazinite said:

> https://groups.google.com/g/endless-sky This is the URL I have when I go to the Google group, instead of URL shown in the issue description. Does this one work?

When I visit the link you posted I get `Content unavailable`.   It also doesn't show up in Google groups searches.

@MCOfficer said:

> Yes, we know. Until MZ reconfigures it somehow, this will have to remain open.

Thanks for acknowledging; I guess I can stop bumping it.  Prior comments seemed to indicate it was isolated to some users and I wanted to emphasize with recent activity on this issue that it's not isolated but more of a general issue for anyone not in the group prior.

I was hoping to participate in community discussion which is why I'm a little noisy.

**Comment by Amazinite** [comment]
> I was hoping to participate in community discussion which is why I'm a little noisy.

The Google group isn't really where discussion goes on anymore. Some people still check it, but most activity is on the Discord server at this point. Invite code is ZeuASSx.

**Comment by Zitchas** [comment]
Just an update re: the google groups. I have noticed recently that the google groups seems to have changed. I don't know if I'm just not seeing a sidebar or something, but now whenever I go there I just get one giant page of discussion threads, and I can't find the sub-forums anymore.

So one way or another, the google groups has become that much less useful or intuitive.

**Comment by quyykk** [comment]
Resolved in #28
